koku/api/organizations/aws/view.py

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright 2020 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""AWS Ogranization views."""
+from api.models import Provider
+from api.organizations.aws.queries import AWSOrgQueryHandler
+from api.organizations.serializers import AWSOrgQueryParamSerializer
+from api.organizations.view import OrganizationView
+
+
+class AWSOrgView(OrganizationView):
+    """AWS Org Base View."""
+
```

Comment:
Do we need to add permissions here? 

For example, here are the permission to view AWS reports:
[https://github.com/project-koku/koku/blob/9607c1dd895fb027d496359f80e8c0601ff0346f/koku/api/report/aws/view.py#L26-L29](https://github.com/project-koku/koku/blob/9607c1dd895fb027d496359f80e8c0601ff0346f/koku/api/report/aws/view.py#L26-L29)

Reply:
@maskarb So I model the org api after the tag api where it is set in the parent class here: https://github.com/project-koku/koku/pull/2062/files#diff-c78a72dda0d219079a94b76ad01950d5R26

However, if we want to set the permissions in the child view instead of the parent it would be a pretty simple switch. Which do you think is better?

Reply:
🤷 to be honest, I'm kinda surprised we AllowAny for the tags views.

I think the question is, if someone only has access to see Azure cost reports, why should they be allowed to see org units in AWS? Maybe @chambridge or @adberglund could weigh in here.

Reply:
I think we should discuss this. One option is another RBAC group for org unit. Options I see

1) limit this on org unit via new RBAC group
2) limit this on existing accounts and only show org units for which the user has account permissions 
3) Some combination of the 2?

Reply:
Just as an update, I am gonna jump on a call later with Andrew and will bring this to confirm which route the permissions should go for the org api.

Reply:
Also, keep in mind this is going into a broader feature branch, so we can merge this jsut fine with the AllowAny permission, and discuss and get the issue written for RBAC control.

Reply:
My $0.02 is that there's an MVP version and a longer-term vision we can aim for.

The MVP, to me, is that org-unit visibility should be restricted to org-admins.

Longer-term, I think we could attempt to map between org-units and RBAC, or we could provide a UI for customers to tell us the mapping we should do. That way we could enable users within the org to view the bits that they ought to have access to.

This may be more of a platform feature than something we want to do ourselves. In that case, I think we could help drive that feature within the platform.

Reply:
I think option (1) makes the most sense to me from the above comment https://github.com/project-koku/koku/pull/2062#discussion_r408998082.

We can broaden it from here, but I don't want to expose organization information through an implicit access to account. I think the intent is really the other direction where you can see AWS account info based on the org unit you are given permission via RBAC to see.

I always envisioned this feature would add us including a `aws.organization` resource type similar to `aws.account` today. And if we get that we would have to do the mapping of org unit hierarchy to collect all the accounts down the tree that user can view data for.